atum/pom.xml

```diff
@@ -35,6 +35,13 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>za.co.absa.commons</groupId>
+            <artifactId>commons_${scala.binary.version}</artifactId>
+            <version>${commons.version}</version>
+            <scope>provided</scope>
```

Comment:
Why is this provided?

Reply:
Thanks for noticing, my mistake, that was a remnant of an exploratory build. Removed.